.github/workflows/build-and-publish-images.yml

```
@@ -359,6 +368,8 @@
           git add common.props
           git add etc/docker/docker-compose*.yml || true
           git commit -m "chore: bump version to $VERSION [skip ci]"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)